Tests/ClientRuntimeTests/NetworkingTests/URLSession/FoundationStreamBridgeTests.swift

```diff
@@ -22,10 +19,7 @@ class FoundationStreamBridgeTests: XCTestCase {
     func test_open_streamsAllDataToOutputBuffer() async throws {
 
         // The maximum size of input streaming data in the tests
-        let maxDataSize = 16_384
-
-        // The max size of the buffer for data being streamed.
-        let maxBufferSize = 2 * maxDataSize
+        let maxDataSize = 65_536  // 64 kb
```

Comment:
Made the data for each test run a little bigger than before.  Has little effect on the time needed to complete tests.